packages/common/graphql/fragments/content-company.js

```diff
@@ -1,8 +1,6 @@
 const gql = require('graphql-tag');
 
-const leadersAlias = process.env.LEADERS_ALIAS || 'leaders-2021';
-
-module.exports = gql`
+module.exports = leadersAlias => gql`
```

Comment:
Arrow functions with a single argument require parentheses
```suggestion
module.exports = (leadersAlias) => gql`
```